nuxeo-core-binarymanager-jclouds/src/main/java/org/nuxeo/ecm/blob/jclouds/JCloudsBinaryManager.java

```diff
@@ -109,6 +113,8 @@ public void initialize(String blobProviderId, Map<String, String> properties) th
             throw new RuntimeException("Missing conf: " + BLOBSTORE_MAP_NAME_KEY);
         }
 
+        endpoint = Framework.getProperty(BLOBSTORE_ENDPOINT_KEY);
```

Comment:
I can do it, but it would break compatibility, so I preferred keeping the old system.